packages/core/src/base-axis-chart.ts

```diff
@@ -133,6 +146,240 @@ export class BaseAxisChart extends BaseChart {
+
+			this.update();
+		};
+
+		const upperDragged = (d) => {
+
+			// Get the cursor's y location.
+			let y = event.y;
+
+			// y must be between the two ends of the line.
+			y = y < y1 ? y1 : y > (sliderBottom - 15) ? (sliderBottom - 15) : y;
+
+			// This assignment is necessary for multiple drag gestures.
+			// It makes the drag.origin function yield the correct value.
+			// Set upper handle position
+			d.y = y;
+
+			// Update the handle location on the slider
+			upperCircle.attr("cy", y);
+
+			// Update axis range
+			this.upperScaleY = 1 - ((y - 25) / 405);
+			sliderTop = y;
+			line.attr("y1", sliderTop + 8);
+
+			this.update();
+		};
+
+		const lowerDragged = (d) => {
+
+			// Get the cursor's y location.
+			let y = event.y;
+
+			// y must be between the two ends of the line.
+			y = y < (sliderTop + 15) ? (sliderTop + 15) : y > y2 ? y2 : y;
+
+			// This assignment is necessary for multiple drag gestures.
+			// It makes the drag.origin function yield the correct value.
+			// Set lower handle position
+			d.y = y;
+
+			// Update the circle location on the slider
+			lowerCircle.attr("cy", y);
+
+			// Update axis range
+			this.lowerScaleY = (y - 25) / 405;
+
+			sliderBottom = y;
+			line.attr("y2", sliderBottom - 8);
+
+			this.update();
+		};
+
+		// Insert the slider element into the action bar. A slider consists of a line and a circle
+		// Bottom handle
+		const svg1 = this.innerWrap.append("svg")
+			.attr("width", width)
+			.attr("height", height)
+			.datum({
+				x: (width / 2) - 240,
+				y: sliderBottom
+			}).call(drag()
+			.on("drag", lowerDragged));
+
+		// Top handle
+		const svg2 = this.innerWrap.append("svg")
+			.attr("width", width)
+			.attr("height", height)
+			.datum({
+				x: (width / 2) - 240,
+				y: sliderTop
+			}).call(drag()
+			.on("drag", upperDragged));
+
+		line = this.innerWrap.append("line")
+			.attr("id", "slider-line")
+			.attr("x1", x - 240)
+			.attr("x2", x - 240)
+			.attr("y1", sliderTop + 8)
+			.attr("y2", sliderBottom - 8)
+			.style("stroke", "red")
+			.style("opacity", 0.5)
+			.style("stroke-linecap", "round")
+			.style("stroke-width", 7)
+			.style("cursor", "grab")
+			.datum({
+				y1: sliderTop,
+				y2: sliderBottom
+			}).call(drag()
+			.on("drag", dragSlider));
+
+		// Make handles draggable
+		lowerCircle = svg1.append("circle")
+			.attr("id", "slider-circle-top")
+			.attr("r", radius)
+			.attr("cy", function(d) { return d.y; })
+			.attr("cx", function(d) { return d.x; })
+			.style("fill", "red")
+			.style("cursor", "ns-resize")
+			.call(drag);
+
+		upperCircle = svg2.append("circle")
+			.attr("id", "slider-circle-bottom")
+			.attr("r", radius)
+			.attr("cy", function(d) { return d.y; })
+			.attr("cx", function(d) { return d.x; })
+			.style("fill", "red")
+			.style("cursor", "ns-resize")
+			.call(drag);
+
+		// Keep chart data elements within the boundaires of the chart
+		const svg = this.innerWrap.append("clipPath")
+			.attr("id", "clip")
```

Comment:
I think it may need to be an ID. It is in this example: https://bl.ocks.org/35degrees/02e5fd2eade71c0a6ffc01d5282def23

Does not work when I use class instead of id